sources/packages/api/src/utilities/credential-type-utils.ts

```diff
@@ -1,9 +1,18 @@
-export function credentialTypeToDisplay(
-  credentialType: string,
-  credentialTypeOther: string,
-): string {
-  if (credentialType?.toLowerCase() === "other") {
-    return credentialTypeOther;
-  }
-  return credentialType;
-}
+/**
+ * Util to get the label of credential type.
+ * @param credentialType
+ * @returns credentialTypeToDisplay
+ */
+export const credentialTypeToDisplay = (credentialType: string): string => {
```

Comment:
I liked the way that the mapping is built but, can we return something in case the mapping is not present, for instance, the _credentialType_ itself. If there is some change in the form.io definition a least this method will return something close to meaningful.